### DISCUSSION TITLE
Can someone confirm this isn't malware?

### DISCUSSION THREAD

**Discussion by skibidisigma101** [baffler/Transparent-Twitch-Chat-Overlay#318] [Q&A]
It's says its unsafe

**Comment by Technetium1** [comment]
That's an entirely generic warning which is not to be taken seriously. See:
- https://github.com/baffler/Transparent-Twitch-Chat-Overlay/issues/274
- https://github.com/baffler/Transparent-Twitch-Chat-Overlay/issues/84
- https://github.com/baffler/Transparent-Twitch-Chat-Overlay/issues/11
- https://github.com/baffler/Transparent-Twitch-Chat-Overlay/issues/17
